tools/loader/ImageClassifier.cpp

```diff
@@ -238,6 +245,51 @@ static void printTopKPairs(const std::vector<FloatIndexPair> &topKPairs) {
   }
 }
 
+/// Convert all placeholders with elements of type \p srcTy into type \p destTy.
+/// Also explicitly convert the input data and the output tensor into the type
+/// \p destTy.
+static void convertPlaceholders(Loader &loader, Context &ctx, Tensor &data,
+                                Placeholder *resultVar, ElemKind srcTy,
+                                ElemKind destTy) {
+  // Convert inputs and outputs into FP16.
```

Comment:
The name of the method is generic, but it does something specific to fp16.

Reply:
At least the comments say fp16.

Reply:
Oops. Good catch. The method itself is not FP16 specific, but I forgot to update the comment. Will fix.